File: vectorex-client-go/res/response_code.go
package res

type ResponseCode struct {
	Code int
	Desc string
}

var (
	Success = ResponseCode{Code: 0, Desc: "SUCCESS"}
	Error   = ResponseCode{Code: 1, Desc: "ERROR"}
	NoAuth  = ResponseCode{Code: 401, Desc: "NO_AUTH"}
)

File: vectorex-client/src/main/java/io/github/javpower/vectorexclient/res/ResponseCode.java
package io.github.javpower.vectorexclient.res;


public enum ResponseCode {

    SUCCESS(0,"SUCCESS"),
    ERROR(1,"ERROR"),

    NO_AUTH(401,"NO_AUTH"),;

    private  final  int code;
    private final  String desc;

    ResponseCode(int code ,String desc){
        this.code = code;
        this.desc =desc;
    }

    public int getCode() {
        return code;
    }

    public String getDesc() {
        return desc;
    }
}
